payment-request/allowpaymentrequest/setting-allowpaymentrequest-timing.https.sub.html

```diff
@@ -18,7 +22,8 @@
   });
 
   window.onmessage = t.step_func_done((e) => {
-    assert_equals(e.data.message, 'Success');
+    assert_equals(e.data.message, 'Exception');
+    assert_array_equals(e.data.details, [true /* ex instanceof DOMException*/, 18 /* ex.code */, 'SecurityError' /* ex.name */]);
```

Comment:
nit: 18/DOMException.SECURITY_ERR/
Then we probably don't need the  /* ex.name */ as what it is is fairly clear.